src/shogun/regression/LeastSquaresRegression.cpp

```diff
@@ -24,4 +18,3 @@ CLeastSquaresRegression::CLeastSquaresRegression(CDenseFeatures<float64_t>* data
 : CLinearRidgeRegression(0, data, lab)
```

Comment:
@Saurabh7 this would be something for you to check/figure out? I think we have seen and solved something similar before. Unless you have ideas @vinx13 ?

Reply:
@karlnapf I only saw this error on CI. I cannot reproduce this locally with docker image.

Reply:
Yes I remember this issue somehow with the same reproducibility issues. Maybe @Saurabh7 remembers

Reply:
I assume this works now?

Reply:
yes, we need to explicit instantiate templates in cpp files to make it linkable

Reply:
Looks like this is solved.

Very Interesting PR btw ! So we are planning to change all APIs to `.fit ` , `.predict` ?

Reply:
Looks like this is solved.

Very Interesting PR btw ! So we are planning to change all APIs to ` .fit `, `.predict` ?

Reply:
Sorry I actually meant to ping @shubham808 ;)
But yes @Saurabh7 we are intending to do that